ambry-router/src/test/java/com.github.ambry.router/GetBlobOperationTest.java

```diff
@@ -52,43 +53,47 @@
 
 /**
  * Tests for {@link GetBlobOperation}
+ * This class creates a {@link NonBlockingRouter} with a {@link MockServer} and does puts through it. The gets,
+ * however are done directly by the tests - that is, the tests create {@link GetBlobOperation} and get requests from
+ * it and then use a {@link NetworkClient} directly to send requests to and get responses from the {@link MockServer}.
+ * Since the {@link NetworkClient} used by the router and the test are different, and since the
+ * {@link GetBlobOperation} created by the tests are never known by the router, there are no conflicts with the
+ * RequestResponseHandler of the router.
+ * Many of the variables are made member variables, so that they can be shared between the router and the
+ * {@link GetBlobOperation}s.
  */
 public class GetBlobOperationTest {
   private static final int MAX_PORTS_PLAIN_TEXT = 3;
   private static final int MAX_PORTS_SSL = 3;
   private static final int CHECKOUT_TIMEOUT_MS = 1000;
-  private static final int BLOB_USER_METADATA_SIZE = 10;
-
+  private final int REPLICAS_COUNT;
```

Comment:
caps are only when a variable is both `static` and `final`. Same for the variable below.